lib/quby/questionnaires/entities/questionnaire.rb

```diff
@@ -270,7 +270,7 @@ def answer_dsl_module
 
                   define_method("#{key}=") do |v|
                     self.value ||= Hash.new
-                    self.value[key] = v
+                    self.value[key] = v.andand.strip
```

Comment:
Omdat onze testopstelling op dit moment is dat de serverkant op dezelfde manier moet kunnen valideren als de clientkant. De test is in het algemeen geschreven, dus 'geeft deze invoer een foutmelding', die test voert hij dan zowel voor het serverdeel als het clientdeel uit. Het is goed mogelijk om de clientkant van de validaties over te slaan, namelijk als op de laatste pagina van de vragenlijst vragen met validaties staan. We willen inderdaad op termijn van de server kant af, maar dat is niet onderdeel van dit ticket.